src/hal/sensor/sensor.cpp

```diff
@@ -1,14 +1,16 @@
 
 #include "../../hal/hal.h"
 #include "../../hal/sensor/sensor.h"
+#include <arduino.h>
```

Comment:
Im just noticing this now, but it should be `Arduino.h` with a capital A. Google suggests that Windows isnt case sensitive in include file names but Linux is, so we should try to avoid that
```suggestion
#include <Arduino.h>
```

Reply:
Fixed!